schema/data/prod/json_schema/milestone.json

```diff
@@ -51,6 +51,11 @@
             "type": "string",
```

Comment:
Sorry, can't comment on lines that weren't changed, but on 43, now that we have a blank option, should we be defaulting to that instead?